src/svc/ctr.rs

```diff
@@ -4,15 +4,17 @@ use crate::{
     res::{parse_result, CtrResult, GenericResultCode, ResultCode},
     Handle,
 };
-use alloc::{vec, vec::Vec};
-use core::{arch::asm, convert::TryInto, ffi::c_void, intrinsics::transmute};
 use ctru_sys::{
     svcAcceptSession, svcContinueDebugEvent, svcCreateEvent, svcCreateMemoryBlock,
     svcDebugActiveProcess, svcExitProcess, svcGetProcessDebugEvent, svcGetProcessInfo,
     svcGetProcessList, svcOpenProcess, svcQueryDebugProcessMemory, svcReadProcessMemory,
     svcReplyAndReceive, svcSignalEvent, svcSleepThread, svcUnmapMemoryBlock,
     svcWaitSynchronization, svcWriteProcessMemory, DebugEventInfo,
 };
+use std::arch::asm;
+use std::convert::TryInto;
+use std::ffi::c_void;
+use std::mem::transmute;
```

Comment:
Let's use this import for consistency:
```suggestion
use std::{arch::asm, convert::TryInto, ffi::c_void, mem::transmute};
```

Reply:
Thoughts on adding the nightly formatting that was added for nex-rs?